packages/template/src/scenes/example.meta

```diff
@@ -3,7 +3,7 @@
   "timeEvents": [
     {
       "name": "circle",
-      "targetTime": 3
+      "targetTime": 0.9940212833664561
```

Comment:
You can't change the `template` package because it's used for testing the rendering.
So any visual/timing change will make the tests fail.

Reply:
_facepalm_, I forgot sorry